web/profiles/custom/yalesites_profile/modules/custom/ys_core/templates/ys-social-links.html.twig

```diff
@@ -0,0 +1,15 @@
+{% if social_links.facebook %}
```

Comment:
See comment above. Since this is already a social links component, the wrapper array only adds ambiguity to the shape of the variable. This can simply be `instagram` as a variable.
https://github.com/yalesites-org/yalesites-project/pull/70/files#r927278582